### ISSUE TITLE
fexist native is broken

### ISSUE THREAD

**Issue by NexiusTailer** [openmultiplayer/open.mp#697]
**Describe the bug**
`fexist` native doesn't work for non-latin names of files in the [latest release](https://github.com/openmultiplayer/open.mp/releases/tag/v1-RC2) and completely broken in the latest [nightly build](https://github.com/openmultiplayer/open.mp/actions/runs/5369293925) i.e. doesn't work even for latin file names.

**To Reproduce**
1. Move any random .txt files into 'scriptfiles' folder
2. Name some of them in cyrillic, for example
3. Put `fexist` check anywhere in gamemode with those names you have for your files
4. Log its return value and see 0 even if the files exist in `scriptfiles` folder

**Screenshots and/or logs**
[nightly build from actions / samp server](https://imgur.com/a/p9MRepz)
[RC2 release / samp server](https://imgur.com/a/YyEOh4Y)

**Release version**
https://github.com/openmultiplayer/open.mp/releases/tag/v1-RC2 / issue with non-latin file names
https://github.com/openmultiplayer/open.mp/actions/runs/5369293925 / issue with any file names

**Comment by Y-Less** [comment]
Latin names seem to work for me.  What are you using exactly?

**Comment by Y-Less** [comment]
(OS and filename)

**Comment by NexiusTailer** [comment]
OS is Windows 10 22H2

Test files can be found here:
[files.zip](https://github.com/openmultiplayer/open.mp/files/12504821/files.zip)

Test script (including .amx, if there are any problems with system/text editor encoding while reproduce it):
[script.zip](https://github.com/openmultiplayer/open.mp/files/12504825/script.zip)

The build from the [latest commit](https://github.com/openmultiplayer/open.mp/actions/runs/6059974527) seems to work properly now, but this still can be useful to reproduce the case yourself to better understand the reasons, if it was fixed just by chance and anything like this could repeat later or there is a similar problem somewhere else with other functions at the moment.

**Comment by dev-karpov0** [comment]
> Latin names seem to work for me. What are you using exactly?

The error caused by not only the filename encoding, but also the path encoding.
The screenshot shows that the file has a latin name and the path to the file contains cyrillic.

**Comment by NexiusTailer** [comment]
> The error caused by not only the filename encoding, but also the path encoding. The screenshot shows that the file has a latin name and the path to the file contains cyrillic.

Oh, that's a good note, I forgot about the path completely. Yeah, this also could've affect the behaviour when even latin filename was not detected as a file which exists. Let me find and re-check the exact commit again when both latin and cyrillic file names didn't work with the path I launched omp server from.

UPD:
In [this nightly build](https://github.com/openmultiplayer/open.mp/actions/runs/5369293925), fexist function doesn't see the files which are actually exist (both latin & cyrillic file names, but the path itself contains cyrillic as well).
In [the release version](https://github.com/openmultiplayer/open.mp/releases/tag/v1-RC2), fexist function doesn't see only the file with any cyrillic name (while latin file name works, even if the path itself contains cyrillic characters).
In [the latest nightly build](https://github.com/openmultiplayer/open.mp/actions/runs/6059974527), fexist function works properly for both file names with any path to them as well.

@dev-karpov0 if the final commits with a fix have already been merged and the cause were completely solved now, let me know if I can close this issue. Thank you.
